test/rubocop/cop/minitest/no_test_cases_test.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class NoTestCases < Minitest::Test
+  def test_registers_offense_for_empty_test_class
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Test class has no test cases.
+      end
+    RUBY
+  end
+
+  def test_registers_offense_for_nonempty_test_class_with_no_test_cases
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Test class has no test cases.
+        def perform
+          1 + 2
+        end
+      end
+    RUBY
+  end
+
+  def test_does_not_register_offense_for_non_test_classes
```

Comment:
It would be good to also have a test, that detects no offenses when the test class has a test case. Something like:

```ruby
class FooTest < Minitest::Test
  def test_foo; end
end
```